git-hub

```diff
@@ -81,21 +137,325 @@ def push():
     "git push user pr/num:branch"
 
     """
+    p = subprocess.Popen("git branch", stdout=subprocess.PIPE)
+    pr = str(p.stdout.read())
+    pr = pr.split("*", 1)[1].split()[0]
+    pr = pr.replace("\\n", "").replace(" ", "").replace("'", "")
     try:
-        with open(".git_hub_cache") as json_file:
-            data = json.load(json_file)
-            other_user = data["user"]
-            pr = data["pr"]
-            branch = data["branch"]
-            p = subprocess.Popen(["git", "push", other_user,
-                                 f'{pr}:{branch}'], stdout=subprocess.PIPE)
-            p.stdout.read()
+        file_path = path_to_git()
+        file_path = os.path.join(file_path, '.git_hub_cache')
+        f = open(file_path, "r")
```

Comment:
Use `with open(...)` instead of open by itself.

Reply:
did not edit this + the above couple edits since they are in teddy's method and I don't want to accidentally mess something up in the push or pr methods